pkg/api/output.go

```diff
@@ -168,6 +177,39 @@ func (instance *Instance) tabOut(columns columnList) *tabwriter.Writer {
 	return w
 }
 
+func (instance *Instance) markdownOut(columns columnList) *tablewriter.Table {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Sign", "Rating"})
```

Comment:
These headings seem incorrect. I'm not suer what `sign` and `rating` are.

Reply:
@sudermanjr great spot! that's a copy and paste error from myself.  

I've removed that now